pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1253,7 +1253,6 @@ func (b *MacvtapBindMechanism) discoverPodNetworkInterface() error {
 		b.virtIface.MAC = &api.MAC{MAC: mac.String()}
 	}
 
-	b.virtIface.MTU = &api.MTU{Size: strconv.Itoa(b.podNicLink.Attrs().MTU)}
```

Comment:
Please return the MTU, we should still pass it.
Other than that I believe the PR is ready to be merged.

Reply:
Done.

Reply:
You have just dropped it once again and I do not understand why the tests have not catch it (there is suppose to be a test for this now).